e2e/tests/ui-driven/src/create-flow.spec.ts

```diff
@@ -114,39 +114,7 @@ test.describe("Flow creation, publish and preview", () => {
     ]);
   });
 
-  test("Cannot preview an unpublished flow", async ({
```

Comment:
E2E tests always start from an empty database and create a flow from scratch which now _includes_ publishing - meaning they no longer have the concept of ever accessing an "unpublished" flow or needing to test initial publish action separately.

The "publishFlow" action is still called throughout this file to save/publish content changes.